### ISSUE TITLE
Error when processing proximal variant

### ISSUE THREAD

**Issue by snowsl** [griffithlab/pVACtools#223]
* pvactools version:1.1.1
* Python version:3.6.5
* Operating System:linux

**Describe the bug**
when i provide a phased vcf file to run pvacseq analysis. VEP used to annotate all vcf file.  Getting a error message:
The wildtype amino acid for variant 10002.CORIN.ENST00000273857.FS.3C/CT with substring MKQSPALAPEER is different than expected.
Actual wildtype amino acid: E
Wildtype amino acid of the proximal_variant: C

**To Reproduce**
pvacseq run total.sort.tarh.vep.readcount.gx.tx.vcf.gz TUMOR HLA-A*11:01,HLA-A*26:01 MHCflurry MHCnuggetsI MHCnuggetsII NNalign NetMHC PickPocket NetMHCcons NetMHCpan SMM SMMPMBEC SMMalign phased/ -e 8,9,10 --normal-sample-name NORMAL -p phased.vep.vcf.gz

**Log Output**
```
The log that gets written to standard out
```

**Output File**
If your bug is related to the contents of an output file, please paste the output file here.

**Expected behavior**

**Comment by susannasiebert** [comment]
This type os error is usually caused of there is a mismatch of build versions and/or ensembl versions at some step during your VEP annotation. 
In order to reproduce this error, you will need to provide the  input total.sort.tarh.vep.readcount.gx.tx.vcf.gz VCF file and the phased VCF? They don't need to contain all of the variants, just the ones that are causing this error.

**Comment by snowsl** [comment]
[phased.cut.vcf.gz](https://github.com/griffithlab/pVACtools/files/2632165/phased.cut.vcf.gz)
[total.cut.vcf.gz](https://github.com/griffithlab/pVACtools/files/2632166/total.cut.vcf.gz)
ok, these two files contain the corresponding error lines. i filter phased vcf file just to keep missense_variant and then the analysis can go on without any problem. But I'm not sure how much impact this mistake will have ?
thanks for you time

**Comment by susannasiebert** [comment]
I made a pull request with a bugfix for this error. Once that is merged in I will make a new hotfix release. I will let you know once that goes live.

**Comment by susannasiebert** [comment]
I just made a new release (1.1.5) that incorporates the fix to this error. Please upgrade to the latest version and give it another try. I'm resolving this issue but feel free to reopen or make a new issue if you're still running into problems.
